docs/assemblies/assembly_prebuilt-servers.adoc

```diff
@@ -0,0 +1,13 @@
+[id='prebuilt-servers_{context}']
+= Pre-built servers
+
+:parent-context: {context}
+
+{Thorntail} has pre-built some https://github.com/thorntail/thorntail/tree/master/standalone-servers[standalone-servers].
+
+include::../modules/ref_prebuilt-microprofile-server.adoc[leveloffset=+1]
+
+include::../modules/ref_prebuilt-keycloak-server.adoc[leveloffset=+1]
```

Comment:
The Keycloak server is not part of product, is maybe `ifndef` this entire include?

Reply:
What about the tagged upstream release ? In fact the same applies to Download links...
Ladislav, @tradej, does a 'product' affect the tagged upstream releases ?

Reply:
My understanding is that in the Thorntail community builds, `product` is never set to `true`, that's just for the downstream RHOAR Thorntail product.

Reply:
OK, I guess I can indeed exclude a top level link to the Keycloak server if it is 'product' but remove the exclusions from the Downloads (should not be a prob if the links in SNAPSHOT docs won't resolve).

Reply:
Yea I think so, too.

Reply:
In Thorntail community, `product` is not defined. In Fabric8 Launcher builds and on the Portal, it is defined.